packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -321,7 +323,7 @@ class _FlexibleSpaceBarState extends State<FlexibleSpaceBar> {
             final bool effectiveCenterTitle = _getEffectiveCenterTitle(theme);
             final EdgeInsetsGeometry padding = widget.titlePadding ??
               EdgeInsetsDirectional.only(
-                start: effectiveCenterTitle ? 0.0 : 72.0,
+                start: effectiveCenterTitle && !(settings.hasLeading ?? false) ? 0.0 : 72.0,
```

Comment:
Shouldn't 72 be based on the FlexibleTitleBar's titlePadding?

Reply:
This is a default when FlexibleTitleBar's titlePadding is not provided. 

If the ` FlexibleTitleBar's titlePadding` to provide, it will override the 72.0 and also bottom padding.